licenses-report.txt

```diff
@@ -2,6 +2,13 @@
 Generated by fossa-cli (https://github.com/fossas/fossa-cli).
 This software includes the following software and licenses:
 
+========================================================================
+Android-Sdk
+========================================================================
+The following software have components provided under the terms of this license:
+
+- Android SDK (from https://www.android.com/)
```

Comment:
SG uses Android SDK? :thinking:

Reply:
At least a dependency or dependency of a dependency does... My guess would be something related to Apollo in the testing modules.